lib/Github/Api/Organization/Actions/Variables.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace Github\Api\Organization\Actions;
+
+use Github\Api\AbstractApi;
+use Github\Exception\MissingArgumentException;
+
+/**
+ * @link https://docs.github.com/en/rest/reference/actions#variables
+ */
+class Variables extends AbstractApi
+{
+    /**
+     * @link https://docs.github.com/en/rest/actions/variables?apiVersion=2022-11-28#list-organization-variables
+     *
+     * @param string $organization
+     *
+     * @return array|string
+     */
+    public function all(string $organization)
+    {
+        return $this->get('/orgs/'.rawurlencode($organization).'/actions/variables');
+    }
+
+    /**
+     * @link https://docs.github.com/en/rest/reference/actions#get-an-organization-secret
+     *
+     * @param string $organization
+     * @param string $variableName
+     *
+     * @return array|string
+     */
+    public function show(string $organization, string $variableName)
+    {
+        return $this->get('/orgs/'.rawurlencode($organization).'/actions/variables/'.rawurlencode($variableName));
+    }
+
+    /**
+     * @link https://docs.github.com/en/rest/actions/variables?apiVersion=2022-11-28#create-an-organization-variable
+     *
+     * @param string $organization
+     * @param array  $parameters
+     *
+     * @throws MissingArgumentException
+     *
+     * @return array|string
+     */
+    public function create(string $organization, array $parameters)
+    {
+        if (!isset($parameters['name'])) {
+            throw new MissingArgumentException(['name']);
+        }
+
+        if (!isset($parameters['value'])) {
+            throw new MissingArgumentException(['value']);
+        }
+
+        if (!isset($parameters['visibility'])) {
+            throw new MissingArgumentException(['visibility']);
+        }
```

Comment:
I would not add this validation in our code, but let the github api validate the request and return any errors related to missing parameters. This way we won't block user in the future if github changes the required parameters